site/src/pages/RoadmapPage/sidebar/MajorRequiredCourseList.tsx

```diff
@@ -37,23 +38,24 @@ const MajorRequiredCourseList: FC = () => {
   const isDark = useContext(ThemeContext).darkMode;
   const isLoggedIn = useIsLoggedIn();
   const majors = useAppSelector((state) => state.courseRequirements.majorList);
-  const specializations = useAppSelector((state) => state.courseRequirements.specializationList);
-  const selectedMajor = useAppSelector((state) => state.courseRequirements.major);
-  const selectedSpec = useAppSelector((state) => state.courseRequirements.specialization);
+  const selectedMajors = useAppSelector((state) => state.courseRequirements.selectedMajors);
 
   const plans = useAppSelector((state) => state.roadmap.plans);
   const planIndex = useAppSelector((state) => state.roadmap.currentPlanIndex);
   const activePlanID = plans[planIndex].id;
 
   /** Must ONLY contain requirements for the selected major/spec. This is used to check whether a major is
    * already loaded, so it must be set to empty if we change the major or spec */
```

Comment:
This comment should be moved to the `courseRequirementsSlice` after `requirements: ProgramRequirement[];` or removed.